src/skydb.ts

```diff
@@ -26,9 +24,8 @@ export async function getJSON(
     method: "get",
     url: this.getSkylinkUrl(skylink),
   });
-  console.log(12);
 
-  return { data: response.data, revision: entry.entry.revision };
+  return response.data;
```

Comment:
Will we still have access to revision ?

Reply:
Yeah, PJ made the point that the registry functions can be called manually if you need the revision. The SkyDB functions should be easy to use without the user having to keep track of the revision. Makes sense to me IMO.

Reply:
Not really, let's say I want to update the content that I just fetched, I will need the revision for that.

Reply:
You would either use the registry from the beginning or just `setJSON`.